### PR TITLE
feat: enable Prometheus Ecto metrics

### PR DESCRIPTION
**Pull request by w3b6x9**
## What kind of change does this PR introduce?

feature

## What is the new behavior?

Enable Prom Ecto metrics

### CODE REVIEW

**Review by abc3 — CHANGES_REQUESTED** [review]
This plugin will provide info about all db hosts, including tenants' databases (migrations use Ecto), which significantly blows up metrics size. We should remove unnecessary fields from this plugin and expose only metrics of the internal db.

**Comment by w3b6x9** [comment]
> This plugin will provide info about all db hosts, including tenants' databases (migrations use Ecto), which significantly blows up metrics size. We should remove unnecessary fields from this plugin and expose only metrics of the internal db.

@abc3 ah, good call. Tenant DB migrations use their own process but under Realtime.Repo. I can just take that out bc Replica.SIN is the same.

File: lib/realtime/monitoring/prom_ex.ex [hunk 1]
```diff
@@ -56,21 +54,20 @@ defmodule Realtime.PromEx do
...
-      # Plugins.Application,
-      {Plugins.Beam, poll_rate: poll_rate, metric_prefix: [:beam]},
-      {Phoenix, router: RealtimeWeb.Router, poll_rate: poll_rate, metric_prefix: [:phoenix]},
-      # {Plugins.Ecto, poll_rate: poll_rate, metric_prefix: [:ecto]},
-      # Plugins.Oban,
-      # Plugins.PhoenixLiveView
-      {OsMon, poll_rate: poll_rate}
+      {Beam, poll_rate: poll_rate, metric_prefix: [:beam]},
+      # list `Realtime.Repo` in `repos` when decoupled from tenant database migrations
+      {Ecto, repos: [Replica.FRA, Replica.IAD, Replica.SIN], metric_prefix: [:ecto]},
```
Comment (w3b6x9) [inline lib/realtime/monitoring/prom_ex.ex:68]: @abc3 `Realtime.Repo` no longer listed.

**Comment by w3b6x9** [comment]
Closing as there appears to be an issue where `Realtime.Repo` metrics are reported even though it's not in the Plugin Ecto `repos` list.

**Pull request closed without merging**